src/components/form/SecuredField.tsx

```diff
@@ -49,15 +49,18 @@ export const SecuredField = <
   if (obj) {
     // @ts-expect-error Grab key from object following naming convention. We check below that we grabbed it correctly.
     const field: SecuredProp<any> = obj[name] ?? obj[name.replace(/Id$/, '')];
-    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- sanity check since we ignored types above
-    if (field.canRead == null || field.canEdit == null) {
-      console.error(
-        `Cannot determine if field should be readable/editable: ${obj.__typename}.${name}`
-      );
-      return null;
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- sanity check
+    if (field) {
```

Comment:
This could be merged into the if statement below.
```tsx
if (!field || field.canRead == null...) {
```

Reply:
I don't think so, because it's setting canRead and canEdit a few linkes outside of the if statement

Reply:
Yes because of the return null statement

Reply:
oops. I get it now. because it can return null when I check for undefined. never mind! My bad.